packages/utils/src/interceptors.js

```diff
@@ -30,6 +30,16 @@ export function interceptor401(error) {
     throw error;
 }
 
+export function interceptor500(error) {
+    if (error.response && error.response.status === 503) {
```

Comment:
The ACK might have been too fast, can we catch all 500 errors here? `if (error.response && error.response.status >= 503) {`

Reply:
@karelhala will saying `500` catch all `5xx` errors? or do I have to mention them individually?